.github/workflows/pr-checker.yml

```
@@ -0,0 +1,31 @@
+name: PR Checker
+
+on:
+  pull_request:
+    types: [opened, synchronize]
```

```suggestion
    types: [opened, ready_for_review, reopened, synchronize]
```
增加reopen和ready for review触发以处理较早的pr